web/src/app/lists/FlatList.tsx

```diff
@@ -162,8 +181,18 @@ export default function FlatList({
   items,
   inset,
   transition,
+  isLoading,
   ...listProps
 }: FlatListProps): JSX.Element {
+  const [displayLoading, setDisplayLoading] = useState(false)
+
+  useEffect(() => {
+    // show loading skeletons after 1 second to avoid load flickering
+    setTimeout(() => {
+      setDisplayLoading(true)
+    }, 1000)
```

Comment:
This could get unmounted during the load and needs to be cleaned up
```suggestion
    // show loading skeletons after 1 second to avoid load flickering
    const t = setTimeout(() => {
      setDisplayLoading(true)
    }, 1000)
    
    return () => clearTimeout(t)
```